projects/humans/skin_animated_humans/worlds/skin_animated_humans.wbt

```diff
@@ -0,0 +1,100 @@
+#VRML_SIM R2021c utf8
+WorldInfo {
+  info [
+    "Skin animation of human models."
+    "In this example BVH motions are played on FBX MakeHuman models."
+    "Author: Guillaume Michel <guillaume.michel.202@gmail.com>"
```

Comment:
@omichel is this email address still valid?

Reply:
I think so.